pages/checkout-beta/_subscription.html

```diff
@@ -0,0 +1,143 @@
+<div class="row">
```

Comment:
Nit: you are not required to shove all the markup into a single file. Jekyll lets you use `include`. Make use off it to keep each file small and readable!